SimG4CMS/HGCalTestBeam/plugins/HGCPassive.cc

```diff
@@ -21,15 +19,18 @@
 
 //#define EDM_ML_DEBUG
 
-HGCPassive::HGCPassive(const edm::ParameterSet &p) : count_(0), init_(false) {
+HGCPassive::HGCPassive(const edm::ParameterSet &p) : topPV_(0), topLV_(0),
+						     count_(0), init_(false) {
 
   edm::ParameterSet m_Passive = p.getParameter<edm::ParameterSet>("HGCPassive");
-  LVNames_  = m_Passive.getUntrackedParameter<std::vector<std::string> >("LVNames");
+  LVNames_   = m_Passive.getUntrackedParameter<std::vector<std::string> >("LVNames");
+  motherName_= m_Passive.getUntrackedParameter<std::string>("MotherName");
```

Comment:
hi @civanch - does this follow the convention other g4 hit tallies to make parameters as untracked? These much change the outputs.